carbon_footprint_node.py

```diff
@@ -15,6 +15,9 @@
 
 from sustainml_py.nodes.CarbonFootprintNode import CarbonFootprintNode
 
+from carbontracker.tracker import CarbonTracker
```

Comment:
I guess that the `carbontracker` should be previously downloaded via`pip` or `apt`, are we including it in the `requirements.txt` of the modules if thats the case ?